### ISSUE TITLE
generator: campus user

### ISSUE THREAD

**Issue by ppanero** [inveniosoftware/invenio-records-permissions#6]
Implement campus user system role (maybe in invenio-access) and the corresponding `Generator`. It might be done with some sort of IP treatment over user defined ranges.

**Comment by fenekku** [comment]
I think this issue https://github.com/inveniosoftware/invenio-access/issues/168, this issue and the current issue are the same. Let's only keep this one open.

**Comment by ppanero** [comment]
> I think this issue #27 and this current issue are the same. Let's only keep this one open.

True, tho the other one was more generic. Sorry I missed the duplicate.
